apps/volunteer/src/pages/shelters/profile/_components/ShelterRecruitments.tsx

```diff
@@ -1,53 +1,98 @@
 import { Box, Card, CardBody, Heading, Text } from '@chakra-ui/react';
+import { useSuspenseInfiniteQuery } from '@tanstack/react-query';
+import { useNavigate, useParams } from 'react-router-dom';
 import ApplicantStatus from 'shared/components/ApplicantStatus';
 import InfoSubtext from 'shared/components/InfoSubtext';
+import useIntersect from 'shared/hooks/useIntersection';
+import { createFormattedTime } from 'shared/utils/date';
+
+import {
+  getRecruitementsOfShelter,
+  RecruitmentOfShleter,
+} from '@/apis/shelter';
+
+function ShelterRecruitmentItem({
+  recruitmentId,
+  recruitmentTitle,
+  recruitmentStartTime,
+  recruitmentDeadline,
+  recruitmentApplicantCount,
+  recruitmentCapacity,
+}: RecruitmentOfShleter) {
+  const navigate = useNavigate();
+
+  const volunteerDate = createFormattedTime(
+    new Date(recruitmentStartTime),
+    'YY.MM.DD',
+  );
+
+  const deadline = createFormattedTime(
+    new Date(recruitmentDeadline),
+    'YY.MM.DD',
+  );
+
+  const goRecruitmentDetail = () => navigate(`/volunteers/${recruitmentId}`);
 
-const DUMMY_RECRUITITEM = {
-  title: '봉사자를 모집합니다.',
-  volunteerDay: '23.12.20',
-  deadline: '23.12.10',
-  applicant: 4,
-  capacity: 10,
-};
-
-type RecruitmentProps = {
-  title: string;
-  volunteerDay: string;
-  deadline: string;
-  applicant: number;
-  capacity: number;
-};
-
-function ShelterRecruitmentItem(recruitmentInfo: RecruitmentProps) {
-  const { title, volunteerDay, deadline, applicant, capacity } =
-    recruitmentInfo;
   return (
-    <Card p={4} pb={3.5} mb={2}>
+    <Card p={4} pb={3.5} mb={2} onClick={goRecruitmentDetail}>
       <CardBody pos="relative" p={0}>
         <Text pb={2} fontWeight={600}>
-          {title}
+          {recruitmentTitle}
         </Text>
-        <InfoSubtext title="봉사일" content={volunteerDay} />
+        <InfoSubtext title="봉사일" content={volunteerDate} />
         <InfoSubtext title="마감일" content={deadline} />
         <Box pos="absolute" right={0} bottom={0}>
-          <ApplicantStatus numerator={applicant} denominator={capacity} />
+          <ApplicantStatus
+            numerator={recruitmentApplicantCount}
+            denominator={recruitmentCapacity}
+          />
         </Box>
       </CardBody>
     </Card>
   );
 }
 
 export default function ShelterRecruitments() {
+  const { id } = useParams();
```

Comment:
```suggestion
  const { id } = useParams<{ id: string }>();
```

타입을 지정해주면 좋을 것 같습니다!

Reply:
넵! 수정하겠습니다